bootstrap.sh

```diff
@@ -21,13 +21,18 @@ if [[ "${RELEASE}" == "master" || "${RELEASE}" == release-0.1 ]]; then
   set +x
   echo "NOTICE: the trace and pack commands are not available from master. RELEASE must be set to a released version (such as v0.2.0). See https://github.com/crossplane/crossplane-cli/releases for the full list of releases." >&2
   set -x
+  curl -sL -o "${PREFIX}"/bin/kubectl-crossplanehttps://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane >/dev/null
```

Comment:
space between `kubectl-crossplane` and `https`?

Reply:
Was force-pushing as you commented :)